tests/README_REGRESSION.md

```diff
@@ -0,0 +1,137 @@
+# Regression Testing for Pastas
+
+## Overview
+
+This directory contains regression testing infrastructure for Pastas. The regression tests verify that model results (`ml.solve()`) remain consistent across different versions of Pastas and its dependencies.
+
+## How it Works
+
+The regression testing uses [`uvtrick`](https://github.com/baggiponte/uvtrick), which allows running code in isolated environments with different package versions using the `uv` package manager.
```

Comment:
This link does not exist?

Reply:
Long live ai.. Link is: https://github.com/koaning/uvtrick